JS/Lesson15/sinan/script.js

```diff
@@ -105,7 +104,21 @@ checkAge(18);
 checkAge(65);
 
 // 3. Write a program that asks the user to enter two numbers and checks if the first number is greater than, less than, or equal to the second number using if else statement.
-
+function checkGreater(numberFirst,NumberSecond) {
+	if (numberFirst < NumberSecond){
+		return 'NumberSecond greater than numberFirst';
```

Comment:
The user of this program does not have to know about the name of the parameters that you used. So try to write more generic messages, something like "Second number is greater that first"